apps/dotcom-worker/src/lib/routes/createRoomSnapshot.ts

```diff
@@ -28,13 +32,9 @@ export async function createRoomSnapshot(request: IRequest, env: Environment): P
 			})),
 			tombstones: {},
 		},
-	}
-
-	const supabase = createSupabaseClient(env)
-	if (!supabase) return noSupabaseSorry()
+	} satisfies R2Snapshot
 
-	const supabaseTable = getSnapshotsTable(env)
-	await supabase.from(supabaseTable).insert(persistedRoomSnapshot)
+	await env.ROOM_SNAPSHOTS.put(getR2KeyForRoom(roomId), JSON.stringify(persistedRoomSnapshot))
```

Comment:
Also used `getR2KeyForRoom` helper. Felt best to be consistent with what we do for rooms.